packages/server/src/sdk/app/views/index.ts

```diff
@@ -178,14 +178,15 @@ function checkRequiredFields(
       continue
     }
 
-    if (!viewSchemaField?.visible) {
+    if (!helpers.views.isCalculationView(view) && !viewSchemaField?.visible) {
```

Comment:
Should this not be done first thing it this function? Or even in the caller, as it's a private method?

Reply:
How come? Is it not relevant to only this conditional?

Reply:
This will be true for any field in a calculation view, will it not? If the view is a calculation one, no required fields are actually needed. Or did I get this wrong?

Reply:
Ohh you're suggesting I skip `checkRequiredFields` entirely for calculation views? That may actually be the correct play, let me check.

Reply:
Done!